<launch>

    <!-- lingao depth camera driver bringup -->
    <include file="$(find lingao_visual)/launch/lingao_depthcamera.launch"/>

    <!-- lingao depth_follower -->
    <node pkg="lingao_follower" name="lingao_depth_follow_node" type="lingao_depth_follow_node" output="screen">

        <param name="enabled" value="true" />  <!-- 启用/禁用追踪 -->
        <param name="x_scale" value="1.0" />   <!--  移动机器人速度的缩放系数 -->
        <param name="z_scale" value="1.2" />   <!-- 旋转机器人速度的缩放系数 -->

        <param name="min_x" value="-0.5" />  <!-- scan box size in meters, horiz from ctr -->
        <param name="max_x" value="0.5" />   <!-- scan box size in meters, horiz from ctr -->
        <param name="min_y" value="0.0" /> <!-- scan box size in meters, distance up from floor -->
        <param name="max_y" value="0.5" /> <!-- scan box size in meters, distance up from floor -->
        <param name="max_z" value="2.0" /> <!-- 框中深度最大值，以上四个字段用来设置框的大小 -->

        <param name="goal_z" value="0.7" /> <!-- 相机离机器人的距离 -->
        
          
    </node>

</launch>
